src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/AutoInsert/HtmlSmartIndentOnAutoInsertProviderTest.cs

```diff
@@ -33,66 +33,66 @@ public static IReadOnlyList<TagHelperDescriptor> TagHelpers
         public void TryResolveInsertion_VoidHtmlTag_Noops()
         {
             RunAutoInsertTest(
-input: @"<input>|<strong></strong>
+input: @"<input>
+$$<strong></strong>
```

Comment:
Isn't this a different test now that `<strong>` is already on the next line?

Reply:
No, the markup processing inserted a character, which in this case was `Environment.NewLine`. The form seen here is the form it was always using, except now you can see it.